app/experimenter/nimbus-ui/src/components/PreviewURL/index.tsx

```diff
@@ -53,8 +69,11 @@ export const PreviewURL: React.FC<
         Preview URL
       </h3>
       <p>
-        <b>Click to copy the link</b> and paste it in your browser. You need to
-        enable <code>nimbus.debug</code> pref before.
+        <b>
+          Click the <code>about:studies</code> link below to copy
+        </b>{" "}
+        then paste it in your browser. Ensure you enable{" "}
```

Comment:
Updated PR with these `{" "}` to correctly space out the text.

Reply:
Looks better now:

![updated text](https://user-images.githubusercontent.com/438537/118012571-121d0700-b306-11eb-8d35-d2c4c71b8c8a.png)